### DISCUSSION TITLE
내일부터 모더나 추가시 진행 가능할까요?

### DISCUSSION THREAD

**Discussion by VCK92** [changdoc/naver-vaccine-macro#102] [Q&A]
먼저 좋은 프로그램 만들어주신 개발자님께 감사인사 드립니다.
내일부터는 모더나랑 화이자 2종으로 진행되는걸로 알고 있는데,
이런 경우 모더나와 화이자를 선택하게 될 거같은데...
이 경우에도 자동신청이 될까요? 기존에는 화이자만 있어서 괜찮았는데... 선택옵션이 생기면 달라지지않을까해서요.

**Comment by ranware** [comment]
아무거나 하시게 되면 잔여백신 나오면 그냥 신청이 되지 않을까요?

**Reply by VCK92** [reply]
과거에 유튜브통해서 했던 매크로는 그 단계에서 막혔다는 이야기가 있어서 여쭈어보았습니다. 결론은 일단 내일 가봐야 알수잇겟네요.
